src/dmd/e2ir.d

```diff
@@ -5953,7 +5956,19 @@ elem *buildArrayBoundsError(IRState *irs, const ref Loc loc)
     auto eassert = el_var(getRtlsym(RTLSYM_DARRAYP));
     auto efile = toEfilenamePtr(cast(Module)irs.blx._module);
     auto eline = el_long(TYint, loc.linnum);
-    return el_bin(OPcall, TYvoid, eassert, el_param(eline, efile));
+    if(upr is null)
+    {
+        upr = el_long(TYsize_t, 0);
+    }
+    if(lwr is null)
+    {
+        lwr = el_long(TYsize_t, 0);
+    }
+    if(elength is null)
+    {
+        elength = el_long(TYsize_t, 0);
+    }
+    return el_bin(OPcall, TYvoid, eassert, el_params(elength, upr, lwr, eline, efile, null));
```

Comment:
What signature does this generate for `_d_arrayboundsp`? Because the druntime PR thinks its something different.